### PR TITLE
Add matomo config and component in onboarding

### PR DESCRIPTION
**Pull request by sandrine-ds**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -20,6 +20,7 @@ ONBOARDING_URL="https://onboarding.swan.local:8080"
 
 # Extra keys for the client
 CLIENT_GOOGLE_MAPS_API_KEY="" # Your google maps API key
+CLIENT_MATOMO_SITE_ID=""
```
Comment (bloodyowl) [inline .env.example:23]: CLIENT_ONBOARDING_MATOMO_SITE_ID

File: clients/onboarding/src/components/OnboardingFooter.tsx [hunk 2]
```diff
@@ -53,30 +54,34 @@ export const OnboardingFooter = ({
...
-              <LakeButton
-                color="gray"
-                mode="secondary"
-                size={large ? "large" : "small"}
-                style={styles.button}
-                onPress={onPrevious}
-              >
-                {t("wizard.back")}
-              </LakeButton>
+              <TrackPressable labelKey={t("wizard.back")}>
```
Comment (bloodyowl) [inline clients/onboarding/src/components/OnboardingFooter.tsx:57]: don't use `t`, the labelKey needs to be stable across locales

File: types/env/index.d.ts [hunk 3]
```diff
@@ -8,4 +8,5 @@ declare const __env: {
   IS_SWAN_MODE: boolean;
   // Client
   CLIENT_GOOGLE_MAPS_API_KEY: string;
+  CLIENT_MATOMO_SITE_ID: string;
```
Comment (bloodyowl) [inline types/env/index.d.ts:11]: ONBOARDING is missing here too

File: clients/onboarding/src/components/OnboardingFooter.tsx [hunk 4]
```diff
@@ -33,52 +39,60 @@ const styles = StyleSheet.create({
 type Props = {
   onPrevious?: () => void;
   onNext: () => void;
-  nextLabel?: string;
+  nextLabel?: TranslationKey;
```
Comment (ncomont) [inline clients/onboarding/src/components/OnboardingFooter.tsx:42]: wdyt about `nextLabelTranslationKey` instead ?

**Review by ncomont — APPROVED** [review]

**Pull request merged**